luigi_pipeline/seqr_loading.py

```diff
@@ -96,7 +96,7 @@ def annotate_old_and_split_multi_hts(self, mt):
         :return: mt that has pre-annotations
         """
         # Named `locus_old` instead of `old_locus` because split_multi_hts drops `old_locus`.
-        return hl.split_multi_hts(mt.annotate_rows(locus_old=mt.locus, alleles_old=mt.alleles))
+        return hl.split_multi_hts(mt.annotate_rows(locus_old=mt.locus, alleles_old=mt.alleles), permit_shuffle=True)
```

Comment:
Could you checck in with the hail team through zulip to confirm this will only try to shuffle when we have the keys out of order with duplicate loci? I know we discussed it and I read the docs as well but I'd like to confirm with them before we merge.